### PR TITLE
Permissions check on components

### PR DESCRIPTION
**Pull request by MihaiIliescu**
Signed-off-by: Iliescu Cristian-Mihai <cristianmihaiiliescu@gmail.com>

### CODE REVIEW

File: src/components/Home/index.tsx [hunk 1]
```diff
@@ -71,10 +74,10 @@ export const links = [
     "link": "/",
     "label": "Home"
   },
-  {
+  ...(hasComponent(VISIBLE_COMPONENTS.ASSET_CATALOG) ? [{
     "link": ASSET_CATALOG_PATH,
     "label": "Catalog"
-  },
+  }] : []),
```
Comment (sarbull) [inline src/components/Home/index.tsx:80]: nice trick you did here 💯

File: src/components/RequirePermissions/index.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  if (hasComponent(props.component)) {
+    return (<>{props.element}</>);
+  } else {
+    if (props.showAccessDenied) {
+      return (<AccessDenied component={props.component}/>);
+    } else {
+      return (<></>);
+    }
+  }
+}
```
Comment (sarbull) [inline src/components/RequirePermissions/index.tsx:14]: don't forget about the '*' for super admin rights

File: src/components/Home/index.tsx (hunk 1 above)
Comment (MihaiIliescu) [reply src/components/Home/index.tsx:80]: thank you

File: src/components/RequirePermissions/index.tsx (hunk 2 above)
Comment (sarbull) [reply src/components/RequirePermissions/index.tsx:14]: i think it applies here https://github.com/odpi/egeria-js-commons/pull/88/files#diff-cdf05ce4e1beabfb807cea0f0ca01f483406aeb8353a109cd003907482175a1dR3

**Review by sarbull — APPROVED** [review]

File: src/components/RequirePermissions/index.tsx (hunk 2 above)
Comment (MihaiIliescu) [reply src/components/RequirePermissions/index.tsx:14]: resolved by "@lfai/egeria-js-commons": "^4.2.16"

**Pull request merged**